File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
	public int solution(int a, int b, int n) {
		int answer = 0;
		int rest = n;
		int num = 0;

		while (rest >= a) {
			num = rest / a * b;
			answer += num;
			rest = rest % a + num;
		}

		return answer;
	}
}

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.kt
class Solution {
    fun solution(a: Int, b: Int, n: Int): Int {
        var answer: Int = 0
        var n = n
        var left = 0

        while (n >= a) {
            left = n / a * b
            answer += left
            n = n % a + left
        }
        return answer
    }
}